src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -280,6 +286,12 @@ databaseChangeLog:
                   defaultValueNumeric: 0
                   constraints:
                     nullable: false
+              - column:
```

Comment:
Please create a new changeset entry for your new column. This process works by only applying change sets to the database that haven't already been run, so this won't be applied on an existing census DB. 😄